[2025-02-08T19:00:38.992+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:00:39.007+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-11-25T00:00:00+00:00 [queued]>
[2025-02-08T19:00:39.019+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-11-25T00:00:00+00:00 [queued]>
[2025-02-08T19:00:39.020+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:00:39.034+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-11-25 00:00:00+00:00
[2025-02-08T19:00:39.043+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=777) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:00:39.044+0000] {standard_task_runner.py:72} INFO - Started process 803 to run task
[2025-02-08T19:00:39.043+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-11-25T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpi1ha0009']
[2025-02-08T19:00:39.045+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask generate_account_dim_data
[2025-02-08T19:00:39.095+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-11-25T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:00:39.196+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-25T00:00:00+00:00'
[2025-02-08T19:00:39.197+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:00:39.218+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T19:00:39.219+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:00:39.227+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:00:39.228+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-11-25T00:00:00+00:00, execution_date=20241125T000000, start_date=20250208T190039, end_date=20250208T190039
[2025-02-08T19:00:39.259+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:00:39.275+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:00:39.277+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:40:37.709+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:40:37.735+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-11-25T00:00:00+00:00 [queued]>
[2025-02-08T20:40:37.744+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-11-25T00:00:00+00:00 [queued]>
[2025-02-08T20:40:37.745+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:40:37.761+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-11-25 00:00:00+00:00
[2025-02-08T20:40:37.773+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1480) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:40:37.777+0000] {standard_task_runner.py:72} INFO - Started process 1502 to run task
[2025-02-08T20:40:37.773+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-11-25T00:00:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmprgier9k3']
[2025-02-08T20:40:37.781+0000] {standard_task_runner.py:105} INFO - Job 219: Subtask generate_account_dim_data
[2025-02-08T20:40:37.824+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-11-25T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:40:37.893+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-25T00:00:00+00:00'
[2025-02-08T20:40:37.894+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:40:37.910+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T20:40:37.911+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:40:37.918+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:40:37.919+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-11-25T00:00:00+00:00, execution_date=20241125T000000, start_date=20250208T204037, end_date=20250208T204037
[2025-02-08T20:40:37.953+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:40:37.970+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:40:37.972+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
